src/components/Capture/Face.js

```diff
@@ -0,0 +1,98 @@
+import { h, Component } from 'preact'
+import { appendToTracking } from '../../Tracker'
+import Selfie from '../Photo/Selfie'
+import Liveness from '../Liveness'
+import Uploader from '../Uploader'
+import Title from '../Title'
+import withPrivacyStatement from './withPrivacyStatement'
+import withCameraDetection from './withCameraDetection'
+import withFlowChangeOnDisconnectCamera from './withFlowChangeOnDisconnectCamera'
+import { isDesktop } from '../utils'
+import { compose } from '../utils/func'
+import { randomId } from '../utils/string'
+import { fileToLossyBase64Image } from '../utils/file.js'
+import CustomFileInput from '../CustomFileInput'
+import style from './style.css'
+
+const defaultPayload = {
+  method: 'face',
+  variant: 'standard',
+  side: null,
+}
+
+class Face extends Component {
+  static defaultProps = {
+    useWebcam: true,
+    requestedVariant: 'standard',
+  }
+
+  componentDidMount() {
+    this.props.useFullScreen(true)
+  }
+
+  handleCapture = payload => {
+    const { actions, nextStep } = this.props
+    const id = randomId()
```

Comment:
FYI Captures don't need an id anymore. I will open a separate ticket to remove it.